io.cpp

```diff
@@ -174,10 +176,9 @@ string IO::simplified( string str )
   bool is_white = false;
   bool was_white = false;
   bool append = false;
-  size_t i = 0;
   size_t n = str.size();
   size_t nm = n - 1;
-  string::iterator it = str.begin();
+  //string::iterator it = str.begin(); ---- not used anywhere.
```

Comment:
why not just deleting it? Same to all similar commenting out in this PR

Reply:
@nixeimar can you please see, I don't know if those variables are of any use in future. So, to be in the safe side I commented out.